frontend/services/gui/src/GUI_Service.hpp

```diff
@@ -84,8 +84,8 @@ class GUI_Service final : public AbstractFrontendService {
     std::vector<std::string> m_queuedProjectFiles;
 
     std::vector<FrontendResource> m_providedResourceReferences;
-    std::vector<FrontendResource> m_requestedResourceReferences;
     std::vector<std::string> m_requestedResourcesNames;
+    std::shared_ptr<frontend_resources::FrontendResourcesMap> frontend_resources;
```

Comment:
There is no strict need to use a `std::shared_ptr` here. Nothing bad happens if `FrontendResourcesMap` gets copied, everything has value semantics.

Reply:
The `FrontendResourceMap` carries data that would be copied to every single window though.